### PR TITLE
Brian.johnson/cms 38 add login authentication

### PR DESCRIPTION
**Pull request by BrianJ-27**
This is to add basic authentication to log into the dashboard upon putting in your username and email. We will make this more secure as we continue the project

### CODE REVIEW

**Review by nrpadilla2 — COMMENTED** [review]
I didn't see the section where you solved the login. Was that something different you were going to show me later?

File: src/login/LoginPage.js [hunk 1]
```diff
@@ -1,49 +1,99 @@
...
+    password: "",
+    isChecked: false,
+  });
+
+  const inputUpdate = (e) => {
+    const target = e.target;
+    const value = target.type === "checkbox" ? target.checked : target.value;
+    const name = target.name;
+    setInputField({ ...inputField, [name]: value, });
+    // setInputField( {...inputField, [e.target.name]: e.target.value} );
```
[the remark this answers is not in the document]
Comment (BrianJ-27) [reply src/login/LoginPage.js]: will do

**Review by BrianJ-27 — COMMENTED** [review]
can you check this out nestor when u get a chance?

**Review by nrpadilla2 — APPROVED** [review]
I noticed that we're using react-router-dom 5.3.0 again. does it work with <router component="xxx" /> ??

I'll approve and we can just do a follow up if it needs a fix.

**Pull request merged**